Tests/L0/SonarQubePostTest/UploadSumamryMdFile.ps1

```diff
@@ -0,0 +1,63 @@
+[CmdletBinding()]
+param()
+
+. $PSScriptRoot\..\..\lib\Initialize-Test.ps1
+. $PSScriptRoot\..\..\..\Tasks\SonarQubePostTest\SonarQubePostTestImpl.ps1
+
+    
+function CreateRandomDir
+{
+   $tempDirectory = [IO.Path]::Combine([IO.Path]::GetTempPath(), [IO.Path]::GetRandomFileName());
+   
+   if ([IO.Directory]::Exists($tempDirectory))
+   {
+       [IO.Directory]::Delete($tempDirectory, $true)
+   }
+   
+   [void][IO.Directory]::CreateDirectory($tempDirectory);
+   return $tempDirectory 
+}
+
+
+#### Test 1 - Uploading fails with a warning if the file is not found
+    
+# Arrange
+Register-Mock GetSonarQubeOutDirectory {"%temp%"}
```

Comment:
It would be safer use a folder you are certain will be empty i.e. one created using CreateRandomDir.